Assets/PatchKit Patcher/Scripts/AppData/Remote/Downloaders/ChunkedHttpDownloader.cs

```diff
@@ -68,8 +76,17 @@ private ChunkedFileStream OpenFileStream()
                 Directory.CreateDirectory(parentDirectory);
             }
 
+            int startChunk = (int) (CalculateContainingChunksRange(_range).Start / _chunksData.ChunkSize);
```

Comment:
Extract CalculateContainingChunksRange() to a variable instead of calling it twice.